### ISSUE TITLE
Plugin Website?

### ISSUE THREAD

**Issue by rdmueller** [asciidoctor/asciidoctor-intellij-plugin#321]
Shall we create a small microsite for the plugin?
We could move the wiki pages to this microsite and thus enable contributors to add to the docs with their PRs.

If you agree, I would suggest to use jBake with Asciidoctor to create this microsite and use netlify to build and host it.

I could do the first PoC.

**Comment by ahus1** [comment]
I'd like to get the structure right before adding too much convent. The personas I have in mind: 

* first time user to AsciiDoc and the plugin ("what can I do here, get me started")
* some experience with AsciiDoc, but maybe new to the plugin ("get my stuff done, be productive")
* power user of both AsciiDoc the plugin ("I want to extend and hack, how to do that")
* contributor ("let's make it better, show me the tools")

jBake - well, don't know much about it besides that it can do AsciiDoctor. And that it can probably use ruby extensions for AsciiDoctor as well, what is nice. Not sure where to get a decent theme from that doesn't look too much vanilla Bootstrap. I wonder if it supports how reload in the browser once a file changes in the file system?

Netlify - is fine, let's agree via direct mail who hosts it from a legal perspective. I'm ok to add me as impressum etc.

part of the main repo vs. separate repo - I'm ok to go single repo. Make jBake part of the gradle build?

**Comment by rdmueller** [comment]
> Not sure where to get a decent theme from that doesn't look too much vanilla Bootstrap.

Point me to any theme (not restricted to bootstrap) and I will create the jBake template :-)

> I wonder if it supports hot reload in the browser once a file changes in the file system?

I don't think we need this when we edit with IntelliJ :-)

**Comment by mojavelinux** [comment]
I'm certainly in support of having a microsite to introduce the plugin and give it a home. I would like to see the documentation for the plugin become part of the new docs.asciidoctor.org site so it's easier to discover for people looking for tools. We're not quite ready to launch docs.asciidoctor.org officially, but you could start getting prepared by moving the documentation to an Antora hierarchy in the code repository. That way, Antora can pick it up and make it part of the docs site (which we can start testing as soon as the content is ready).

**Comment by rdmueller** [comment]
@mojavelinux is there somewhere a preview so that we can build the docs in the same way the other docs are build?

**Comment by mojavelinux** [comment]
Here's the site in progress: https://github.com/asciidoctor/docs.asciidoctor.org

**Comment by rdmueller** [comment]
I would like to start with this task during #hacktober

First I would like to create a WIP branch with some documents copied from the wiki and one or two rendering mechanisms.
I will start to work on this in my own fork and will send a first PR asap.
But I don't know how to open a new branch in this repository - maybe through the PR...

**Comment by mojavelinux** [comment]
> But I don't know how to open a new branch in this repository - maybe through the PR...

A PR can't introduce a branch (though I really wish it could). The branch can be created from the web UI, which you can then target using a PR.

**Comment by ahus1** [comment]
@rdmueller - I'm looking forward to a first draft! If you create a branch in a clone of this repo, create a PR and check the box "allow edits from maintainers" we can collaborate on that branch. Once it is ready, it can be merged to the master of this repo (depending on the setup of the site). 

If you think a branch in this repo would make collaboration easier, please let me know.

**Comment by ahus1** [comment]
@rdmueller - the discussion about a site kind of re-started in #479. 
Over the last months I've worked with Antora a bit more, and would like to see Antora for the content plus something else for the landing page.

**Comment by ahus1** [comment]
Closing as superseded by #479
